src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -544,20 +544,6 @@ public override Process[] InputObject
 
         #region Overrides
 
-        /// <summary>
-        /// Check the elevation mode if IncludeUserName is specified.
-        /// </summary>
-        protected override void BeginProcessing()
-        {
-            // The parameter 'IncludeUserName' requires administrator privilege
-            if (IncludeUserName.IsPresent && !Utils.IsAdministrator())
-            {
-                var ex = new InvalidOperationException(ProcessResources.IncludeUserNameRequiresElevation);
```

Comment:
The resource string `ProcessResources.IncludeUserNameRequiresElevation` needs to be removed.

Reply:
Thanks will remove it.